src/Discord.Net.Rest/Entities/Channels/RestTextChannel.cs

```diff
@@ -15,6 +15,8 @@ public class RestTextChannel : RestGuildChannel, IRestMessageChannel, ITextChann
 
         public string Mention => MentionUtils.MentionChannel(Id);
 
+        internal bool Nsfw { get; private set; }
```

Comment:
Why internal? Additionally, IChannel.IsNsfw has a different behaviour to this; surely it may become confusing?